src/schema/users.ts

```diff
@@ -567,6 +648,44 @@ export const resolvers: IResolvers<any, Context> = {
         )}.png?r=${randomStr}`,
       };
     },
+    updateUserProfile: async (
```

Comment:
I couldn't find where you handle the file upload of the image

Reply:
Kindly check if the recently pushed commit for file upload fits. Thank you!